src/routes.php

```diff
@@ -1,13 +1,21 @@
 <?php
+use \Slim\Http\Request as Request;
+use \Slim\Http\Response as Response;
 
 //FrontEnd
 $app->get('/','HomePageController');
 $app->get('/admin', 'AdminController');
 $app->get('/register', 'RegisterController');
-$app->get('/addPartner', function ($request, $response, $args) {
+$app->get('/addHiringPartner', function ($request, $response, $args) {
     return $this->renderer->render($response, 'addHiringPartner.phtml', $args);
 });
```

Comment:
Is there a reason you didn't do this in a dedicated controller?

Reply:
This was done for testing.  Setting up the controller has been done in another task.